Rakefile

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/rake
+require 'yaml'
+require 'json'
+require 'net/http'
+require 'uri'
+require 'plist'
+
+desc 'Bootstrap this repository for development'
+task :bootstrap do
+  [:StencilSwiftKit, :SwiftGenKit, :SwiftGen, :templates].each do |repository|
+    next if Dir.exists? "#{repository}"
+
+    sh "git clone git@github.com:SwiftGen/#{repository}.git --recursive"
+    Dir.chdir("#{repository}") do
+      sh "git submodule update --init --recursive"
+    end
+  end
+end
```

Comment:
Should we also have a `cp` namespace with tasks like `rakelib` and `gitignore` to copy those files from Eve to the repos? Or do we expect to change them so little that it wouldn't be worth a task for that?